vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -348,7 +350,7 @@ def set_relationship_on_hosts(hosts)
       #
 
       def normalize_blank_property_num(property)
-        property.try(:to_i)
+        property.try(:to_i) || 0
```

Comment:
we can discuss here about what makes sense. https://github.com/ManageIQ/manageiq/issues/3307

but things like there is hash instead of num and string has > 255 chars, cases should be handled. I saw that data of different OpenStack installations can differ like this (even on same version) 

let me know what you think